Capstone/All Comments/DetailCommentCell.m

```diff
@@ -0,0 +1,108 @@
+//
+//  DetailCommentCell.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 8/8/22.
+//
+
+#import "DetailCommentCell.h"
+
+@implementation DetailCommentCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+}
+
+- (void)setComment:(Comment *)comment {
+    self.usernameLabel.text = comment.usernameForDisplaying;
+    self.userCommentLabel.text = comment.comment;
+    self.counterLabel.text = [[NSString alloc] initWithFormat:@"%d", comment.vote];
+    self.userProfileImageView.file = comment.image;
+    [self.userProfileImageView loadInBackground];
+    self.currentUser = [PFUser currentUser];
+    self.commentObjectId = comment.objectId;
+    [self voteStatusChecker];
+    [self getTheCommentFromParse];
+}
+
+- (void)getTheCommentFromParse {
+    PFQuery *query = [PFQuery queryWithClassName:@"Comments"];
+    [query whereKey:@"objectId" equalTo:self.commentObjectId];
+    [query findObjectsInBackgroundWithBlock:^(NSArray *comments, NSError *error) {
+        if (comments != nil) {
+            self.commentFromParse = [comments objectAtIndex:0];
+            self.upvotes = self.commentFromParse[@"upvoted"];
+            self.downvotes = self.commentFromParse[@"downvotedUsers"];
```

Comment:
my comment about this got deleted somehow, but i mentioned it in my review summary so thought i should just mention it here again: wouldn't we want `upvoted` and `downvotedUsers` be Relations instead of an array of strings?

for instance, if we wanted to show all the posts a user upvoted and downvoted, Relations would make this more effective to do than strings; plus, strings require additional parsing that we shouldn't necessarily need